site/blog/unlisted/beta.md

```diff
@@ -39,7 +39,7 @@ Support for multiple AWS regions with default region behavior in part because we
 
 ### 🪄 Automagically, please
 
-We redid the UX to allow handling infrastructure changes automatically and wrapping delicate, or complex, changes within a special IaSQL transaction akin to transactions in a regular database. This means no more `apply`. Do you want to programmatically modify your infra or control plane? We got you!
+We redid the UX to allow handling infrastructure changes automatically and wrapping delicate, or complex, changes within a special [IaSQL transaction](/docs/transaction) akin to transactions in a regular database. This means no more `apply`. Do you want to programmatically modify your infra or control plane? We got you!
```

Comment:
Most people reading this are not going to know what "This means no more `apply`." means. We could probably drop that sentence?